packages/hardhat-plugin-ethers/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import "@nomicfoundation/hardhat-ignition";
```

Comment:
What are the implications (if any) of potentially importing this twice? i.e. if a user has already imported the base plugin in their Hardhat config file.

Is this something we can/should check for?

Reply:
I think this is now taken care of with the check for other ignition plugins.
I will run a manual test to see.